{"author": {}, "body": "[deleted]", "created_utc": 1586211898.0, "distinguished": null, "id": "fmmtpwl", "is_submitter": false, "link_id": "t3_fuy9mr", "parent_id": "t1_fmi9340", "score": 1, "stickied": false, "subreddit_id": "t5_2sjnq", "downs": 0, "total_awards_received": 0, "edited": false, "name": "t1_fmmtpwl", "collapsed": false, "permalink": "/r/TechNewsToday/comments/fuy9mr/no_5g_didnt_cause_the_coronavirus_pandemic/fmmtpwl/"}